docs/CQ.rst

```diff
@@ -0,0 +1,59 @@
+Create a Contribution Questionnaire
+===================================
+
+All the new dependencies and all the dependencies updates should be submited as a new CQ into the
+`ECLIPSE IP Zilla <https://dev.eclipse.org/ipzilla/query.cgi>`__.
+
+Submitting a CQ
+---------------
+
+Visit `https://projects.eclipse.org/projects/locationtech.geotrellis/ <https://projects.eclipse.org/projects/locationtech.geotrellis/>`__
+and log into the system.
+
+.. figure:: img/locationtech-geotrellis.png
+   :alt: The GeoTrellis project page
+
+On the right, in the ``COMMITTER TOOLS`` side bar, you can find a link `Create a Contribution Questionnaire <https://projects.eclipse.org/projects/locationtech.geotrellis/cq/create>`__
```

Comment:
It might be good to circle or emphasize the location of the link on the page to make it easier to see.